src/main/java/net/patchworkmc/manifest/accesstransformer/v2/TransformedClass.java

```diff
@@ -42,23 +46,47 @@ protected TransformedClass(String name, Finalization finalization, AccessLevel a
 		return Collections.unmodifiableSet(methods);
 	}
 
-	@Override
-	public TransformedClass remap(Remapper remapper) {
+	public TransformedClass remap(Remapper remapper, Consumer<MissingMappingException> errorLogger) throws MissingMappingException {
 		String remappedName = remapper.remapClassName(getName());
 		Set<TransformedField> remappedFields = new HashSet<>();
 		Set<TransformedMethod> remappedMethods = new HashSet<>();
 
 		for (TransformedField field : this.getFields()) {
-			remappedFields.add(field.remap(remapper));
+			try {
+				remappedFields.add(field.remap(remapper));
+			} catch (MissingMappingException ex) {
+				errorLogger.accept(ex);
+			}
 		}
 
 		for (TransformedMethod method : getMethods()) {
-			remappedMethods.add(method.remap(remapper));
+			try {
+				remappedMethods.add(method.remap(remapper));
+			} catch (MissingMappingException ex) {
+				errorLogger.accept(ex);
+			}
 		}
 
 		return new TransformedClass(remappedName, getFinalization(), getAccessLevel(), remappedFields, remappedMethods, fieldWildcard, methodWildcard);
 	}
 
+	/**
+	 * @deprecated Use {@link TransformedClass#remap(Remapper, Consumer)} instead.
+	 */
+	@Override
+	@Deprecated
+	public TransformedClass remap(Remapper remapper) throws MissingMappingException {
+		LinkedList<MissingMappingException> suppressedExceptions = new LinkedList<>();
```

Comment:
Why not just use an ArrayList?
```suggestion
		List<MissingMappingException> suppressedExceptions = new ArrayList<>();
```